tools/windows-node-installer/pkg/openshift/util.go

```diff
@@ -0,0 +1,43 @@
+package openshift
```

Comment:
This file can live inside the `client` pkg.

Reply:
do you think we should move this to `client` package or move `client` package to this `openshift` package?

Reply:
Both are valid but I prefer the former. @ravisantoshgudimetla can be the tiebreaker.

Reply:
Yeah, client package is fine for now, once we notice, we need lots of methods for the client struct, we can move it to its own package.